src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -1502,6 +1502,51 @@ public static IEnumerable<string> ParseConditionalCompilationSymbols(string valu
             return defines.AsEnumerable();
         }
 
+        public static bool TryParseLanguageVersion(string value, out LanguageVersion version)
```

Comment:
@jasonmalinowski in this branch, we're just doing the bare minimum to test the ref readonly feature on the new version. @jcouv's work (he can correct me if I'm wrong) will handle the versioning story, with all the side items required (project system/IDE), and it will eventually be merged into this branch, overwriting the tiny fixes here.